doc/src/build/fullnode.md

```diff
@@ -93,10 +93,24 @@ in how a fullnode is run.
 0. *Prerequisite* Before beginning ensure that the following tools are
    installed in your environment:
     - Rust toolchain managed by [rustup](https://rustup.rs/)
+        - install by `curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh`
     - `git`
-    - `cmake`
-
-1. Set up your fork of the Sui repository:
+    - [cmake](https://cmake.org/install/)
+1. (Optional) If you are using Linux, it's likely you want to install extra dependencies. For example, in Ubuntu, run
+```
+    apt-get update \
+    && DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install -y --no-install-recommends \
+    tzdata \
+    git \
+    ca-certificates \
+    curl \
+    build-essential \
+    libssl-dev \
+    pkg-config \
+    libclang-dev \
+    cmake
+```
```

Comment:
These instructions were intentionally written to avoid the problem of needing to enumerate how to install all dependencies for every platform. Because of that I don't think its best to include this here. Maybe a better option would be to link to the "installing sui" page.

Reply:
ie maybe we just include a link to here: https://docs.sui.io/build/install#prerequisites and then we can use that as the single place for enumerating that.

Reply:
All of the same requirements exist for the `sui` binary as the `sui-node` crate is a dependency of the `sui` binary

Reply:
> All of the same requirements exist for the `sui` binary as the `sui-node` crate is a dependency of the `sui` binary

I'm happy to add these details to the central Install docs. But we still haven't said anything about recommended operating systems as requested in:
https://github.com/MystenLabs/sui/issues/1367

Can we recommend Linux/Ubuntu? Do we support macOS or Windows? Let's settle this there.

Reply:
I just want to add we are using Ubuntu/Debian distros for current development.  We'll add other distros later.  PRs welcomed of course.

Reply:
I'm including this in fullnode now to get the doc out with minimal disruption to the rest of the docs:
https://github.com/MystenLabs/sui/pull/2566

Reply:
Aye, we can move this info later as appropriate